README.md

```diff
@@ -116,11 +116,11 @@ _Usage examples are based on a project created with [Angular CLI](https://github
   }
   ```
   
-3. Use `<datepicker>` with `formControlName` in `app.component.html`
+3. Use `<ng2-datepicker>` with `formControlName` in `app.component.html`
   
   ```html
   <form [formGroup]="dataForm">
-    <datepicker formControlName="date" [expanded]="true"></datepicker>
+    <ng2-datepicker formControlName="date" [expanded]="true"></ng2-datepicker>
```

Comment:
@gaaamii thank you for this! can you remove `[expanded]` here and elsewhere, its not a valid input anymore.